onnxruntime/test/python/onnxruntime_test_python_azure.py

```diff
@@ -1,107 +1,14 @@
-import unittest
-
-import numpy as np
-from helper import get_name
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
 
+import unittest
 import onnxruntime as ort
 
 
-class TestAmlEndpoint(unittest.TestCase):
```

Comment:
Why remove so many tests?

Reply:
The tests are moved to ort extension.
During ort packaging, we do not load extension, meaning this is no way to test inferencing.